service/src/fusedev.rs

```diff
@@ -205,6 +206,66 @@ impl FusedevFsService {
         session.wake().map_err(NydusError::SessionShutdown)?;
         Ok(())
     }
+
+    fn notify_resend_by_fusedev(&self) -> NydusResult<()> {
+        let session = self.session.lock().unwrap();
+        let file = session
+            .get_fuse_file()
+            .ok_or(NydusError::FuseFileNotFound)?;
+
+        let fd = file.as_raw_fd();
+        let mut buf = vec![0u8; session.bufsize()];
+        let writer: FuseDevWriter = FuseDevWriter::new(fd, &mut buf).unwrap();
+
+        self.server
+            .notify_resend(writer)
+            .map_err(NydusError::FuseWriteError)?;
+        Ok(())
+    }
+
+    fn notify_resend_by_sysfs(&self) -> NydusResult<()> {
+        // Notify the kernel to resend the request by writing to `/sys/fs/fuse/connections/<cid>/resend`
```

Comment:
It is better to check if the file exists first, and then write to it, otherwise kernels that do not support this file will throw an error.